src/Model/Filter/Finder.php

```diff
@@ -37,6 +38,21 @@ public function process(): bool
         foreach ($map as $to => $from) {
             $args[$to] = $args[$from] ?? null;
         }
+        $casts = $this->getConfig('cast');
+        foreach ($casts as $field => $toType) {
+            $value = $args[$field] ?? null;
+            if ($value === null) {
+                continue;
+            }
+
+            if (is_callable($toType)) {
```

Comment:
```suggestion
            if ($toType instanceof \Closure) {
```
I highly dislike `is_callable()` and doubt anyone will use anything besides a closure here.